public/diagnostic.js

```diff
@@ -23,3 +24,21 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+const order = Ember.Object.extend({
+  // unitPrice:
```

Comment:
These are self-referential in Ember objects, so Ember knows how to make the getters/setters.
`unitPrice: this.unitPrice,`